tools/sharkbait/src/sharkbait/permissions.clj

```diff
@@ -21,9 +22,10 @@
   "Sets the allowed actions for a permission definition."
   [permission-def]
   (let [delegate (.getAttributeDefActionDelegate permission-def)]
-    (.configureActionList delegate ["read" "write" "own"])
-    (add-implied-action delegate "own" "write")
-    (add-implied-action delegate "write" "read"))
+    (.configureActionList delegate [read write own originator])
+    (add-implied-action delegate originator own)
```

Comment:
Won't this cause issues once we want to use this to allow labels-only editing, or am I missing the intention?

I'd assumed this was the solution for the problem that came up in the meeting yesterday, that originators can still edit labels even after the app is published but not the rest -- but, with this line they'd always have ownership permission, so it wouldn't limit them and we'd have to remove the permission.

I'd figured we'd do something like set an explicit permission for the limited public editing we'll still allow, plus an originator thing like this, and set up the inheritance with two branches: own -> write -> limited write -> read as well as originator -> limited write; that way, we grant both ownership and originator to the originator and we can still remove all ownership and write permissions when an app is published and the originator retains their limited write permission.

But I suspect I'm misunderstanding :)